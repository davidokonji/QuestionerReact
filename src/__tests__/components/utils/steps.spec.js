import React from 'react';
import { shallow } from 'enzyme';
import Steps from '../../../components/utils/steps';

test('should render step component correctly', () => {
  const wrapper = shallow(<Steps />);

  expect(wrapper).toMatchSnapshot();
});
