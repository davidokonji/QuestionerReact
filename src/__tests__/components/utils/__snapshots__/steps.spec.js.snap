// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render step component correctly 1`] = `
<div
  className="steps"
>
  <Step
    image="register.svg"
    text="Register as a Questioner user"
  />
  <Step
    image="sign-in.svg"
    text="Login as a User"
  />
  <Step
    image="conference.svg"
    text="You can view Meet Ups and RSVP for any meetup"
  />
  <Step
    image="asking.svg"
    text="Ask Question? any information concerning a Meet Up"
  />
</div>
`;
